src/services/completions.ts

```diff
@@ -1322,6 +1322,12 @@ namespace ts.Completions {
             const moduleSpecifierSymbol = typeChecker.getSymbolAtLocation(importOrExportDeclaration.moduleSpecifier);
             if (moduleSpecifierSymbol) {
                 exports = typeChecker.getExportsOfModule(moduleSpecifierSymbol);
+                const exportEquals = typeChecker.resolveExternalModuleSymbol(moduleSpecifierSymbol);
+                if (exportEquals !== moduleSpecifierSymbol) {
+                    // Location doesn't matter so long as it's not an identifier.
+                    const exportEqualsType = typeChecker.getTypeOfSymbolAtLocation(exportEquals, moduleSpecifier);
+                    exports = ts.concatenate(exports, typeChecker.getPropertiesOfType(exportEqualsType));

```

Comment:
Like in the test file, you could have both `var Foo` and `namespace Foo`. `exports` (first set) is for `namespace Foo`, `getPropertiesOfType` is for `var Foo`.